andela/helios-ah-frontend
# Protect authenticated components

#### What does this PR do?
This PR implements functionality that protects authenticated component from been viewed by unauthenticated users.
 
#### Description of Task to be completed?
- Created a HOC component that redirects unauthenticated users to login component when they try to access authenticated components

#### How should this be manually tested?
- cd to project directory
- run npm install
- navigate to /create-article route
- it should redirect you back to login page if you were not logged in.

#### What are the relevant pivotal tracker stories?
N/A

#### Any background context you want to add?
None

#### Screenshots
N/A
